ui improvement of sidebar

ARJUN-SABU opened the pull request:
issue #3 :
Improved UI of side bar by:
1. Adding blue border to the playground title
2. Adding change of background-color of ListItem component when it is clicked showing that which component was clicked.
3. Changed inline css to css modules for footer and sidebar.

tewarig reviewed (commented):
well done..
![Screenshot from 2021-03-11 14-54-18](https://user-images.githubusercontent.com/54894783/110765439-2dbe4f80-827a-11eb-90f3-7beaa3079ca7.png)
I was thinking it would be great if the playground and 🚀 comes under one line only. 
![Screenshot from 2021-03-11 15-01-17](https://user-images.githubusercontent.com/54894783/110766259-fbf9b880-827a-11eb-80c3-005ba36b75b3.png)

you did great work keep it up.

ARJUN-SABU commented:
Oh thanks! But, when I am viewing the deployed version myself then the title and the rocket are on the same line and in your case its on different lines 🤔.

pranshuchittora commented:
> well done..
> ![Screenshot from 2021-03-11 14-54-18](https://user-images.githubusercontent.com/54894783/110765439-2dbe4f80-827a-11eb-90f3-7beaa3079ca7.png)
> I was thinking it would be great if the playground and  comes under one line only.
> ![Screenshot from 2021-03-11 15-01-17](https://user-images.githubusercontent.com/54894783/110766259-fbf9b880-827a-11eb-80c3-005ba36b75b3.png)
> 
> you did great work keep it up.

It's coming in one line, I think are zoomed-in kindly reset the page zoom level to 100%

pranshuchittora commented on src/containers/Drawer/ResponsiveDrawer.module.css:
```diff
@@ -0,0 +1,5 @@
+.brand_name {
+  font-weight: 100;
+  padding: 8px;
+  border-bottom: 3px solid #2089dc;
```
We can replace this with a <Divider> from mui/core

ARJUN-SABU replied:
Ok, I will add those styles from css module to makeStyles of material-ui/core/styles

ARJUN-SABU commented:
1. Changed the css module styles of title to material-ui/core/styles format
2. Added blue coloured border-right when hovering over side elements and selecting any one of them, indicating which element was selected.

<img width="1440" alt="The side bar" src="https://user-images.githubusercontent.com/63174870/110881250-0c0aaa00-8306-11eb-88ca-4adeeb8c0b30.png">

pranshuchittora commented on src/containers/Drawer/index.jsx:
```diff
@@ -57,13 +57,31 @@ const useStyles = makeStyles((theme) => ({
+    fontWeight: "100",
+    padding: "8px 0",
+    marginLeft: "5px",
+    borderBottom: "3px solid #2089dc",
```
Pls remove this & replace with a <Divider>

ARJUN-SABU replied:
Ok removing it.

ARJUN-SABU commented:
<img width="1440" alt="SideBar update_2" src="https://user-images.githubusercontent.com/63174870/110960390-b6211b00-8374-11eb-911a-0a7e715e5867.png">

Removed the above mentioned code.
Is it ok?

pranshuchittora commented:
Thanks :)
